website/content/docs/commands/roles/create.mdx

```diff
@@ -33,6 +33,8 @@ $ boundary roles create [options] [args]
 
 - `-description=<string>` - The description to assign to the role.
 - `-grant-scope-id=<string>` - The scope ID for any grants set on the role.
+The `grant-scope-id` field is now deprecated in favor of multiple grant scope support.
```

Comment:
I think we should link directly to the new method of doing this via the CLI, too

```suggestion
The `grant-scope-id` field is now deprecated in favor of multiple grant scope support using [add-grant-scopes](/boundary/docs/commands/roles/add-grant-scopes).
```